po/fr.po

```diff
@@ -1,5 +1,723 @@
+"X-Generator: Poedit 2.4.1\n"
+"X-Poedit-Basepath: .\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"X-Poedit-SearchPath-0: .\n"
+
+#: src/Dialogs/ExportDialog.vala:151
+msgid "Export to:"
+msgstr "Exporter vers :"
+
+#: src/Dialogs/ExportDialog.vala:157
+msgid "Select Folder"
+msgstr "Sélectionner un dossier"
+
+#: src/Dialogs/ExportDialog.vala:166
+msgid "Format:"
+msgstr "Format :"
+
+#: src/Dialogs/ExportDialog.vala:179
+msgid "Quality:"
+msgstr "Qualité :"
+
+#: src/Dialogs/ExportDialog.vala:191
+msgid "Compression:"
+msgstr "Compression :"
+
+#: src/Dialogs/ExportDialog.vala:205
+msgid "Transparency:"
+msgstr "Transparence :"
+
+#: src/Dialogs/ExportDialog.vala:218
+msgid "Scale:"
+msgstr "Échelle :"
+
+#: src/Dialogs/ExportDialog.vala:242 src/FileFormat/FileManager.vala:49
+#: src/FileFormat/FileManager.vala:105
+msgid "Cancel"
+msgstr "Annuler"
+
+#: src/Dialogs/ExportDialog.vala:249 src/Dialogs/ShortcutsDialog.vala:63
+#: src/Layouts/HeaderBar.vala:112
+msgid "Export"
+msgstr "Exporter"
+
+#: src/Dialogs/ReleaseDialog.vala:47 src/Dialogs/SettingsDialog.vala:217
+msgid ""
+"WARNING!\n"
+"Akira is still under development and not ready for production. Missing "
+"features, random bugs, and black holes opening in your kitchen are to be "
+"expected."
+msgstr ""
+"ATTENTION !\n"
+"Akira est encore en cours de développement et n'est pas prêt à être mis en "
+"production. Il faut s'attendre à ce que des fonctionnalités soit "
+"manquantes, qu'il y ait des bugs aléatoires et que des trous noirs "
+"s'ouvrent dans votre cuisine."
+
+#: src/Dialogs/ReleaseDialog.vala:100 src/Dialogs/SettingsDialog.vala:249
+msgid "Make a Donation"
+msgstr "Faire un don"
+
+#: src/Dialogs/ReleaseDialog.vala:109 src/Dialogs/SettingsDialog.vala:258
+msgid "Suggest Translations"
+msgstr "Suggérer des traductions"
+
+#: src/Dialogs/ReleaseDialog.vala:118 src/Dialogs/SettingsDialog.vala:267
+msgid "Report a Problem"
+msgstr "Signaler un problème"
+
+#: src/Dialogs/SettingsDialog.vala:40
+msgid "Preferences"
+msgstr "Préférences"
+
+#: src/Dialogs/SettingsDialog.vala:50 src/Dialogs/SettingsDialog.vala:73
+#: src/Dialogs/ShortcutsDialog.vala:45
+msgid "General"
+msgstr "Général"
+
+#: src/Dialogs/SettingsDialog.vala:51 src/Dialogs/SettingsDialog.vala:86
+msgid "Interface"
+msgstr "Interface"
+
+#: src/Dialogs/SettingsDialog.vala:52 src/Layouts/HeaderBar.vala:280
+msgid "Shapes"
+msgstr "Formes"
+
+#: src/Dialogs/SettingsDialog.vala:53
+msgid "About"
+msgstr "À propos"
+
+#: src/Dialogs/SettingsDialog.vala:74
+msgid "Auto Reopen Latest File:"
+msgstr "Rouvrir automatiquement le dernier fichier :"
+
+#: src/Dialogs/SettingsDialog.vala:88
+msgid "Use Dark Theme:"
+msgstr "Utiliser le thème sombre :"
+
+#: src/Dialogs/SettingsDialog.vala:92
+msgid "Invert Panels Order:"
+msgstr "Inverser l'ordre des panneaux :"
+
+#: src/Dialogs/SettingsDialog.vala:96
+msgid "Restart application to apply this change."
+msgstr "Redémarrer l'application pour appliquer ce changement."
```

Comment:
If it's not a button but an action asked to the user, I think we should use the imperative instead of the infinitive

> Redémarrez l'application pour appliquer ce changement.